test/integration/singlecluster/scheduler/podsready/scheduler_test.go

```diff
@@ -780,15 +780,21 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReadyNonblockingMode", func() {
 				util.ExpectWorkloadsToHaveQuotaReservation(ctx, k8sClient, standaloneClusterQ.Name, wl1)
 				// To avoid flakiness, we don't verify if the workload has a QuotaReserved=false with pending reason here.
 			})
+			ginkgo.By("waiting for the first workload to be evicted again (second eviction)", func() {
+				util.AwaitWorkloadEvictionByPodsReadyTimeout(ctx, k8sClient, client.ObjectKeyFromObject(wl1), podsReadyTimeout)
+				util.FinishEvictionForWorkloads(ctx, k8sClient, wl1)
+			})
```

Comment:
I have a question why adding this block. 

I fully agree it is interesting to test after the second eviction, but it overwrites the previous asserts.
So, I'm wondering it might be better to assert on requeueAt before this ginkgo.By block and after.